hdrop/hdrop

```diff
@@ -26,45 +35,76 @@ print_help() {
   echo "support for the following flags in the [COMMAND] string:"
   echo ""
   echo " -a         ('foot' terminal emulator)"
-  echo " --class    (all other programs)."
+  echo " --class    (all other programs)"
   echo ""
   echo "See man page for more information"
 }
 
 print_version() {
-  echo "hdrop version: 0.2.4"
+  echo "hdrop version: 0.2.5"
 }
 
 notify() {
   notify-send "$@" || hyprctl notify 0 5000 0 "$@"
+  echo "$@"
+}
+
+notify_low() {
+  notify-send -u low "$@" || hyprctl notify 1 5000 0 "$@"
+  echo "$@"
 }
 
 BACKGROUND=""
+INSENSITIVE=false
+VERBOSE=false
 
-case "$1" in
-"")
-  notify "hdrop: missing Argument" "Run 'hdrop -h' for more information"
-  print_help
-  exit 1
-  ;;
--h | --help)
-  print_help
-  exit
-  ;;
--V | --version)
-  print_version
-  exit
-  ;;
--b | --background)
-  BACKGROUND="hyprctl dispatch -- exec [workspace special:hdrop silent] "
-  shift
-  ;;
--*)
-  notify "hdrop: unrecognized flag" "Run 'hdrop -h' for more information"
-  print_help
-  exit 1
-  ;;
-esac
+# shopt -s extglob
+
+while true; do
+  case "$1" in
```

Comment:
I think this is getting long enough that `getopt` would be better suited for parsing arguments.

Reply:
I thought about that but then wasn't sure how to separate hdrop flags from the flags of the program to be run by hdrop. How would you do that?

Reply:
There's a general consistency between programs I use that if you pass `--`, flags after that are no longer the calling program's business. Maybe something like that can be handled.

Reply:
I think i found a solution...